apps/prairielearn/assets/scripts/lib/mathjax.ts

```diff
@@ -9,6 +9,13 @@ declare global {
 }
 
 const mathjaxPromise = new Promise<void>((resolve, reject) => {
+  if (window.MathJax) {
+    // Something else already loaded MathJax on this page. Just resolve the promise
+    // once MathJax reports that it is ready.
+    window.MathJax.startup.promise.then(() => resolve());
```

Comment:
Nit, should this also handle a reject?
```suggestion
    window.MathJax.startup.promise.then(resolve, reject);
```

Reply:
Yep, good call, done in 7c6024c and tested locally.